frontend/package.json

```diff
@@ -26,19 +26,20 @@
     "marked": "^2.0.0",
     "material-ui-chip-input": "^1.1.0",
     "query-string": "^6.13.1",
+    "rc-slider": "^9.7.1",
     "react": "^16.13.1",
+    "react-avatar-editor": "^11.1.0",
```

Comment:
Can you ensure you're running npm 6 (not npm 7), then undo the changes to package-lock.json, then redo them? It appears that because you're using npm 7, the lockfile version has changed to 2, but we want to keep it at version 1 for now (at least until we can upgrade default versions of node / npm on CI at a later time).